app/models/facility.rb

```diff
@@ -126,8 +126,8 @@ def complete_problem_order_details
 
   def set_journal_mask
     f = Facility.all.order(journal_mask: :desc).first
-    self.journal_mask = if f && f.journal_mask.match(/^C(\d{2})$/)
-                          sprintf("C%02d", Regexp.last_match(1).to_i + 1)
+    self.journal_mask = if f&.journal_mask&.match(/^C(\d{2})$/)
```

Comment:
Fun fact about this I learned recently: it means we can never have more than 100 facilities.